### PR TITLE
Flatex PDF Importer

### PR DESCRIPTION
**Pull request by stoeggich**
support for new transactions
close  issues #6003 

Push #5978 first, I will adjust this here again later.

### CODE REVIEW

**Comment by stoeggich** [comment]
@kimmerin please check Bond fix

File: name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/FinTechGroupBankPDFExtractor.java [hunk 1]
```diff
@@ -139,11 +139,7 @@ private void addBuySellTransaction()
                             // @formatter:on
                             if (v.get("notation") != null && !v.get("notation").startsWith("St"))
                             {
-                                var shares = asBigDecimal(v.get("shares"));
-                                t.setShares(Values.Share.factorize(shares.doubleValue() / 100));
-
-                                if (t.getPortfolioTransaction().getType().isPurchase())
-                                    type.getCurrentContext().putBoolean("isPurchaseBonds", true);
+                                t.setShares(asBondNominal(v.get("shares")));
```
[the remark this answers is not in the document]
Comment (stoeggich) [reply name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/FinTechGroupBankPDFExtractor.java:142]: The isPurchaseBonds flag was intentionally removed together with the two Zinsbetrag fee sections in the same commit — there is no remaining code that reads it, so restoring it would only add dead state. Booking accrued interest as a fee was exactly what this change fixes; it is now part of the purchase price, and for foreign-currency bonds it is included in the gross value unit. Equivalence of asBondNominal() with the previous doubleValue() / 100 conversion was verified for all nominal values in the test documents and additional edge cases, with identical results.

**Comment by stoeggich** [comment]
Push #5978 first, I will adjust this here again later.
